src/test/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ find_package(GTest REQUIRED)
 #
 # caffe2plan
 #
-if(${TRITON_ENABLE_TENSORRT})
+if(${TRITON_ENABLE_TENSORRT} AND NOT ${TRITON_IGPU_BUILD})
```

Comment:
Are we going to fee our `build.py` with extra CMake argument. 
I wasn't able from glance find it.

Reply:
I think @nv-kmcgill53 is right that we cannot build this part as it involves some peculiar linking. It used to be there I think.

Reply:
@mc-nv We are currently adding this to the build. [See here](https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/blob/3fe14c296226d262d5299c516f589ba5bd564a20/.gitlab-ci.yml#L1126)